x/sequencer/keeper/sequencer.go

```diff
@@ -10,6 +9,9 @@ import (
 	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
 )
 
+// used to indicate that no sequencer is available for a proposer / next proposer role
+const NO_SEQUENCER_AVAILABLE = ""
```

Comment:
the sentinel only makes sense if you use it with direct value comparison, not len(x)==0